tools/generator/src/DTO/SchemeAutogenerationMode.swift

```diff
@@ -0,0 +1,5 @@
+enum SchemeAutogenerationMode: String, Decodable {
```

Comment:
Is `top_level` coming later?

Reply:
Yes. I created a separate TODO. Adding it was more than just plumbing the value through and adding a simple `guard`. I figured that it deserved its own PR.